home/static/home/explore-page.css

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Danit, I made a 'CSS' folder inside the static/home. I think it will be more organized if you move this file there,
sorry for not notifying you about it earlier

Reply:
@liornoy 
I moved it to the css folded